frontend/src/metabase/public/components/widgets/AdvancedSettingsPane.jsx

```diff
@@ -112,11 +113,17 @@ const AdvancedSettingsPane = ({
   );
 };
 
-const Section = ({ className, title, children }) => (
-  <div className={cx(className, "mb3 pb4 border-row-divider border-medium")}>
-    {title && <h3>{title}</h3>}
-    {children}
-  </div>
-);
+const Section = ({ className, title, children }) => {
+  const sectionId = useUniqueId();
+  return (
+    <section
```

Comment:
This change is used in `frontend/src/metabase/public/components/widgets/EmbedModalContent.unit.spec.tsx`